relay-spans/src/span.rs

```diff
@@ -473,14 +564,81 @@ mod tests {
                     "value": {
                         "stringValue": "my 1st transaction"
                     }
+                },
+                {
+                    "key": "sentry.measurements.memory.value",
+                    "value": {
+                        "doubleValue": 9001.0
+                    }
+                },
+                {
+                    "key": "sentry.measurements.memory.unit",
+                    "value": {
+                        "stringValue": "byte"
+                    }
+                },
+                {
+                    "key": "sentry.metrics_summary.some_metric",
```

Comment:
Every metric name in the metrics summary becomes its own attribute to reduce nesting. We cannot flatten this any further because the value of this attribute is an array.

Reply:
This looks good to me. https://opentelemetry.io/docs/specs/otel/common/attribute-type-mapping/ is marked as experimental, though I don't see why any 3rd party vendors would emit Sentry specific attributes 🙂